### PR TITLE
chore(unsplash.avatar): deprecate

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
Is listed in #1044.

This PR deprecates `faker.image.unsplash.avatar()`  by m,making the following changes:
- use internal `deprecated()` function
- adding the `@deprecated` JSDoc parameter 
- adding a test case for the deprecation log

### CODE REVIEW

**Comment by xDivisionByZerox** [comment]
Is this a `refactor` or `chore`?

**Comment by import-brain** [comment]
> Is this a `refactor` or `chore`?

Most of the previously merged deprecation PRs have been `chore`, so I'm leaning towards that.

File: src/modules/image/providers/unsplash.ts [hunk 1]
```diff
@@ -33,9 +34,16 @@ export class Unsplash {
    * @example
    * faker.internet.avatar()
    * // 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/315.jpg'
+   *
+   * @deprecated
```
Comment (ST-DDT) [inline src/modules/image/providers/unsplash.ts:38]: Please add `@see faker.image.avatar`

File: src/modules/image/providers/unsplash.ts [hunk 2]
```diff
@@ -33,9 +34,16 @@ export class Unsplash {
...
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with unsplash.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.unsplash.avatar()',
+      proposed: 'faker.image.avatar()',
+      since: 'v7.2.0',
```
Comment (ST-DDT) [inline src/modules/image/providers/unsplash.ts:44]: The next version would be 7.3.0.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: src/modules/image/providers/unsplash.ts (hunk 1 above)
Comment (xDivisionByZerox) [reply src/modules/image/providers/unsplash.ts:40]: done dac19f3

File: src/modules/image/providers/unsplash.ts (hunk 2 above)
Comment (xDivisionByZerox) [reply src/modules/image/providers/unsplash.ts]: done 23d9c4c

File: src/modules/image/providers/unsplash.ts [hunk 3]
```diff
@@ -30,12 +31,21 @@ export class Unsplash {
...
    * faker.internet.avatar()
    * // 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/315.jpg'
+   *
+   * @deprecated
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with unsplash.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.unsplash.avatar()',
+      proposed: 'faker.image.avatar()',
```
Comment (xDivisionByZerox) [inline src/modules/image/providers/unsplash.ts:45]: I'm just seeing that the actual implementation is faker.internet, not faker.image... gonna fix this

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

File: test/image.spec.ts [hunk 4]
```diff
@@ -216,6 +216,22 @@ describe('image', () => {
       });
     });
 
+    describe('avatar()', () => {
```
Comment (Shinigami92) [inline test/image.spec.ts:219]: @ST-DDT same question as in the other PR (test for deprecation message needed?)

Comment (ST-DDT) [reply test/image.spec.ts:219]: As mentioned in the other PR.
I consider the deprecation check covered by the generic `test-examples-and-deprecations` check, so the extra test is optional and I wouldn't add it myself.

Comment (Shinigami92) [reply test/image.spec.ts:219]: Yeah, let's revert this file as I would say we don't need to pollute our tests with that anymore

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: test/image.spec.ts (hunk 4 above)
Comment (xDivisionByZerox) [reply test/image.spec.ts]: done 4b74651

**Review by import-brain — DISMISSED** [review]

**Review by Shinigami92 — APPROVED** [review]

File: src/modules/image/providers/unsplash.ts [hunk 5]
```diff
@@ -30,12 +31,21 @@ export class Unsplash {
...
+   *
+   * @deprecated
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with unsplash.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.unsplash.avatar()',
+      proposed: 'faker.internet.avatar()',
+      since: 'v7.3.0',
+      until: 'v8.0.0',
```
Comment (ST-DDT) [inline src/modules/image/providers/unsplash.ts:47]:
```suggestion
      since: '7.3',
      until: '8.0',
```

Changed in preparation #1067 .

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**